recipe/cgo/test.sh

```diff
@@ -16,8 +16,11 @@ go env
 # Run go's built-in test
 case $(uname -s) in
   Darwin)
+    if [[ $(uname -m) != arm64 ]]; then
+        export CONDA_BUILD_SYSROOT=/opt/MacOSX10.14.sdk
+    fi
```

Comment:
Why?

Reply:
was present on the anaconda side, probably wrong for our needs